vshard/router/init.lua

```diff
@@ -796,6 +799,28 @@ local function router_cfg(router, cfg, is_reload)
     end
 end
 
+--
+-- Synchronize update of router configuration from mutiple fibers.
+-- At each point of time only single fiber can execute 'router_cfg' for particular router.
+-- Other fibers are waiting for current call of 'router_cfg' to return.
```

Comment:
Please, keep comments inside 0-66 columns. Besides being part of our code style (and in picodata as well), it also really helps to read, when you do it 8-12 hours per day. Please, check all other places you added too.